pennylane/_qubit_device.py

```diff
@@ -1886,3 +1886,138 @@ def adjoint_jacobian(self, tape, starting_state=None, use_device_state=False):
                 bras[kk, ...] = self._apply_operation(bras[kk, ...], adj_op)
 
         return jac
+
+    def adjoint_jacobian_new(self, tape, starting_state=None, use_device_state=False):
+        """Implements the adjoint method outlined in
```

Comment:
Let's temporarily make a comment that this is the version of `adjoint_jacobian` for the new return type system.